src/components/game/Debug.js

```diff
@@ -0,0 +1,62 @@
+import { FILLED_COLOR } from "./graphics/Theme";
+
+export const DEBUG = true;
+export const MESSAGE_SIZE = 24;
+export const MESSAGE_OFFSET = 10;
+
+// TODO: Add more args to debugCell like x and y position or left/right align to render more of these
+export function debugCell(canvas, cell, x0, y0, boardX, boardY) {
```

Comment:
Use an iterator

Reply:
added iterator alternatives but commented them out, they are available for when someone wants to work more on that but the current one is a quick-and-sloppy impl of smth that reads really nicely. I'd rather have that for now